From 4807eff98b411d403da474fa62f973aa2412b203 Mon Sep 17 00:00:00 2001
From: Paul Reioux <reioux@gmail.com>
Date: Mon, 25 Nov 2013 20:59:26 -0600
Subject: [PATCH 478/507] intellidemand: remove sampling rate updates based on
 suspend

this is causing a rare race condition where every once in a while, the phone
will not wake up properly from a suspend/resume cycle.  Disable for now
until I have more time to debug this properly

Signed-off-by: Paul Reioux <reioux@gmail.com>
Signed-off-by: Simarpreet Singh <simar@linux.com>
---
 drivers/cpufreq/cpufreq_intellidemand.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/cpufreq/cpufreq_intellidemand.c b/drivers/cpufreq/cpufreq_intellidemand.c
index 8ea39ff..214f00c 100644
--- a/drivers/cpufreq/cpufreq_intellidemand.c
+++ b/drivers/cpufreq/cpufreq_intellidemand.c
@@ -29,7 +29,7 @@
 #include <linux/workqueue.h>
 #include <linux/slab.h>
 
-#ifdef CONFIG_POWERSUSPEND
+#if 0
 #include <linux/powersuspend.h>
 #endif
 
@@ -77,7 +77,7 @@
 #define SUP_HIGH_SLOW_UP_DUR (5)
 #define SUP_FREQ_LEVEL (14)
 
-#ifdef CONFIG_POWERSUSPEND
+#if 0
 static unsigned long stored_sampling_rate;
 #endif
 
@@ -2318,7 +2318,7 @@ bail_acq_sema_failed:
 	return 0;
 }
 
-#ifdef CONFIG_POWERSUSPEND
+#if 0
 static void cpufreq_intellidemand_power_suspend(struct power_suspend *h)
 {
 	mutex_lock(&dbs_mutex);
@@ -2402,7 +2402,7 @@ static int __init cpufreq_gov_dbs_init(void)
 							 "dbs_sync/%d", i);
 	}
 
-#ifdef CONFIG_POWERSUSPEND
+#if 0
 	register_power_suspend(&cpufreq_intellidemand_power_suspend_info);
 #endif
 	return cpufreq_register_governor(&cpufreq_gov_intellidemand);
-- 
1.7.9.5

